<!DOCTYPE html>
<html lang="en-US">
     <head>
          <title>CRISPR</title>

          <meta charset="UTF-8">
          <meta name="viewport" content="width=device-width, initial-scale=1.0">
          <meta name="description" content="TechBio, Genomics, Sequencing, Synthetic Biology, and iPSC">
          <meta name="author" content="CoderKid2k">

          <link rel="stylesheet" href="../styles/header.css">
          <link rel="stylesheet" href="../styles/main.css">
          <link rel="stylesheet" href="../styles/footer.css">

     </head>
     <body>

          <!--There Starts the Header-->

          <header id="header">

               <div class="brand-title">
                    <h1 class="site-name"><a href="../index.html">Biotech2k.com</a></h1>
               </div>

               <nav>
                    <ul class="menu">
                         <li class="nav-item">
                              <a href="../index.html">Home</a>
                         </li>
                         <li class="nav-item">
                              <a href="../pages/innovation.html">Innovation</a>
                         </li>
                         <li class="nav-item">
                              <a href="../pages/companies.html">Companies</a>
                         </li>
                         <li class="nav-item">
                              <a href="../pages/science.html">Science</a>
                         </li>
                         <li class="nav-item">
                              <a href="../pages/clinicaldata.html">Clinical Data</a>
                         </li>
                    </ul>
               </nav>

               <div class="hamburger">
                    <div class="chevron"></div>
                    <div class="chevron"></div>
                    <div class="chevron"></div>
               </div>

          </header> 

          <!--Here Starts the Main Body-->

          <main>

               <h1 class="science-headers">CRISPR</h1>

               <div class="science-container">

                    
                    <ul>
                         <li class="science-topics">
                              <a href="#intro">Intro to CRISPR</a>
                         </li>
                         <li class="science-topics">
                              <a href="#types">Types of CRISPR</a>
                         </li>
                         <li class="science-topics">
                              <a href="#cas9">CAS9</a>
                         </li>
                         <li class="science-topics">
                              <a href="#cas12">CAS12</a>
                         </li>
                         <li class="science-topics">
                              <a href="#mad7">MAD7</a>
                         </li>
                         <li class="science-topics">
                              <a href="#casx">CASX</a>
                         </li>
                         <li class="science-topics">
                              <a href="#cas13">CAS13</a>
                         </li>
                         <li class="science-topics">
                              <a href="#mini">CAS14 and Phi</a>
                         </li>
                         <li class="science-topics">
                              <a href="#base">Base Editing</a>
                         </li>
                         <li class="science-topics">
                              <a href="#prime">Prime Editing</a>
                         </li>
                         <li class="science-topics">
                              <a href="#paste">PASTE Editing</a>
                         </li>
                    </ul>
                    

                    <section>
                         <h2 class="science-headers" id="intro">Intro to CRISPR</h2>
                         <p>     
                              What does CRISPR stand for? It stands for Clustered Regularly Interspaced Short Palindromic Repeats. 
                              This was discovered by scientists studying bacteria. Bacteria get infected by viruses called 
                              Bacteriophages. They attach to the bacteria and inject their viral DNA. Scientists noticed that the 
                              bacterial genome had a sequence of repeats. It showed a small sequence of DNA then a repeat followed by 
                              another small sequence and another repeat. They named this area of the bacteria genome the CRISPR region. 
                              This is the bacteria's immune system.
                         </p>
                         <p>
                              It works through type 1 CRISPR enzymes like CAS1 and CAS2, chopping up the bacteria DNA. Then a small 
                              sequence of the viral DNA called the protospacer is inserted into the CRISPR region of the bacteria 
                              genome. What happens next is very awesome. The bacteria transcribes the entire CRISPR region into one 
                              long messenger RNA called the pre-transcript. This gets chopped up so each DNA sequence that matches 
                              the virus' protospacer becomes the CRISPR RNA (crRNA) and the repeat becomes the trans activating 
                              RNA (tracrRNA). The repeat segments become the tracrRNA that will get loaded into an enzyme called CAS9. 
                              CAS9 stands for CRISPR Associated Protein 9. The CAS9 will recognize the tracrRNA and bind to it.
                              The crRNA will then bind to the tracrRNA to make a complete guide RNA. The two RNA segments bind together 
                              by complementary base pair binding by hydrogen bonding.  Once the CAS9 is loaded with the complete guide 
                              RNA, it will go and find any DNA that has the matching protospacer for the loaded guide. The guide is a complete 
                              complementary sequence of RNA that matches that protospacer in the Viral genome.
                         </p>
                         <p>
                              The actual guide sequence is about 20 nucleotides long. Once it finds the matching viral DNA, the 
                              CAS9 enzyme will cut that DNA. This destroys the viral DNA.  I know what you are thinking. If the DNA 
                              sequence copied into the CRISPR region of the Bacteria matches the viral DNA, how does the CAS9 not 
                              cut up that bacteria's DNA too? The answer to that is the PAM sequence of the CAS9 enzyme. This stands 
                              for Protospacer Adjacent Motif. This is a small segment of nucleotides in the DNA that is recognized 
                              by the CAS9 enzyme which is N-G-G. The N stands for any nucleotide while the G stands for Guanine. This 
                              basically means the CAS9 enzyme not only needs to match the guide RNA sequence to the protospacer, but 
                              it also needs to match its PAM to the 2 Guanines near the protospacer. If both don't exist, the CAS9 
                              will not cut. As you can guess, the CRISPR region of the bacteria genome would not contain the PAM 
                              sequence so the CAS9 would be selective for targeting the viral DNA, but not the bacteria's DNA.
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="types">Types of CRISPR</h2>
                         <p>     
                              The CRISPR system was discovered in 2012 by two scientists by the names of Jennifer Doudna and Emmanuelle 
                              Charpentier. While studying the CRISPR immune systems of the bacteria Streptococcus Pyogenes, they 
                              discovered a system that could be adapted for human genome editing. CRISPR stands for Clustered Regularly 
                              InterSpaced Palindromic Repeats. It turned out this system was the immune system for the bacteria. 
                         </p>
                         <p>
                              The actual CRISPR enzymes that are adapted for human genome editing are called CRISPR Associated 
                              (CAS) Enzymes. The first one discovered was CAS9. These enzymes contain nucleases which are domains 
                              that allow for DNA cutting. The CAS9 has 2 of these nuclease domains to cut both strands of DNA.
                              The CAS9 enzyme is guided by a strand of RNA that matches to the desired sequence in the DNA. 
                              This becomes a search function for finding and cutting the DNA at a specific location. Since the discovery 
                              of the first CAS9 nuclease, scientists and companies have been out there searching the microbe world for 
                              new CRISPR systems and nuclease enzymes. We are going to look at them all.
                         </p>
                         <p>
                              CAS enzymes are broken down into two classes with Class 1 enzymes and Class 2 enzymes. These two classes 
                              are further broken down into Types and Subtypes for the CAS enzymes. This gets pretty complicated so I 
                              will give you the general differences. The Class 1 CAS enzymes are made up of multiple units called subunits. 
                              They have to work together to function. One such example is CAS3 which works with CAS8 and CAS11 to 
                              edit RNA. Class 2 CAS enzymes are one large structure that contains everything in one. This includes many of the 
                              current CAS enzymes like CAS9, CASX, CAS12, CAS13 and MAD7. All of these come from different microbes but 
                              work similarly.
                         </p>
                         <p>
                              All of them contain the RuvC nuclease domain to cut the DNA or RNA. The CAS9 is the only one I know that 
                              also contains the NHN nuclease domain as a second nuclease. CAS enzymes are further broken down into what their nuclease cuts. 
                              Some of them are designed to cut RNA, and they are being built for RNA editing like CAS13. Most are designed for 
                              DNA editing like CAS9, CAS12, MAD7 and CASX.
                         </p>
                         <p>
                              The major advance in CAS enzyme was the discovery of other CAS enzyme in other microbes. There has been an 
                              explosion of new CAS enzymes that we can use to develop editing tools. CAS9 was the original discovered in Streptococcus 
                              Pyogenes. Not long after Cpf1 (now known as CAS12) was discovered in 2015. This quickly grew in popularity. 
                              Then came the discovery of CASX in E-coli from Berkeley  which is now used by Scribe Therapeutics. MAD7 was discovered in a 
                              bacteria in Madagascar which gave it the name MAD7. It is now used by Inscripta, and they license it to 
                              many companies. This works very much like CAS12 in its function. Both CAS14 and CAS-Phi were discovered by Mammoth Bioscience 
                              and are being developed by them. These are much smaller versions of the CAS enzymes. I call them the CAS 
                              mini. Both of them are about half the size of CAS9. CAS13 is the first RNA editor being developed by the Zhang lab. It was discovered 
                              from a bacteria called Leptotrichia wadei. The last one is CAS11 which is part of the CAS3 system. It is 
                              being develop for applications in gene knockout and possibly RNA editing.
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="cas9">CAS9</h2>
                         <p>     
                              The CAS9 system comes with 2 components. The first is the guide RNA and the second is the CAS9 enzyme itself. The 
                              CAS9 is made up of a few key domains with the two nucleases that are the HNH and RuvC domains. Both the nucleases will 
                              activate and make a double stranded break at the exact same location in the DNA on both strands. Some CAS9 
                              enzymes have one of these 2 nucleases mutated to be inactive. These are called a Nickase as they will only 
                              cut 1 strand of DNA. The next key domain of the CAS9 is the PAM sequence. This matches a specific sequence of the DNA. 
                              Each CAS enzyme has a different PAM site which it recognizes. The CAS9 PAM will recognize the sequence 
                              N-G-G. This stands for Anything-Guanine-Guanine.
                         </p>
                         <p>
                              The CAS9 nucleases will cut about 3 to 4 bases away from its PAM sequence. This is important as this short 
                              distance makes it difficult for CAS9 to do multiple edits. Once an edit occurs, the site for the guide and 
                              PAM matching will be disrupted by the edit.
                         </p>
                         <p>
                              The last part of the CAS9 system is the guide RNA (gRNA). If you recall, the wild type guides where made
                              up of 2 segements in the crRNA and the tracrRNA. They would come together to form the complete guide RNA.
                              This would be too complex for use in gene editing so they use a single guide RNA which basically links 
                              them together. This is called a single guide RNA (sgRNA) and just basically makes the 2 different guides into 
                              one guide. They do this with a little linker loop at the end of the 2 guides to attach them.
                         </p>
                         <p>
                              The sgRNA is installed in the delivery vector as RNA, but the CAS9 is often installed into the vector as a 
                              messenger RNA. Some of the newer RNP packaging will include the CAS9 as a full enzyme. When its used as a mRNA, 
                              it is about 4,100 bases in length and encodes a enzyme about 1,368 amino acids long. This is a very big 
                              package for most traditional vectors. Most companies have moved toward LNP technology for delivery.
                              The CAS9 will load the guide RNA when inside the cell. It enters the nucleus where it will bind to the DNA. 
                              The CAS9 opens the DNA and runs the 20 base guide along the DNA until it finds its match. Once the guide 
                              and PAM have a match, the nucleases will cut the DNA.
                         </p>
                         <p>
                              This system can be used for Disruptions, Deletions or 
                              Insertions of genes. The disruption is done by causing Double Stranded Breaks at the target site to cause 
                              mutations that prevent reading of that gene. This effectively knocks out the gene. This works really well 
                              in-vivo as we have some very strong clinical data so far. The second use is gene deletion. This can be created 
                              using 2 CAS9 enzymes along with 2 guides to attempt to cut and remove an entire section of the DNA. I don't 
                              think this has ever been attempted in-vivo, and I don't think I would want it to be. The last use is gene insertion 
                              using homology directed repair using a template strand that gets included with the CAS9 and guide RNA at 
                              the time of delivery. The CAS9 does the DNA cutting and the template strand guides the repair.
                         </p>
                         <p>
                              The biggest concern 
                              with the CAS9 system is the Double Stranded Breaks. When DNA is cut like this, it can repair in unexpected 
                              ways. You can end up with insertions or deletions (Indels) of bases.
                              The other big concern is chromosomal rearrangements when used for multiple edits at once. 
                              Typically they have to do an edit and run the cell through the cell cycle to ensure the first 
                              edit was good before they can do the next.
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="cas12">CAS12</h2>
                         <p>     
                              The CAS12 system comes with 2 components. The first is the guide RNA and the second is the CAS12 
                              enzyme itself. The CAS12 is made up of one key domain with a single cutting nuclease with RuvC. When 
                              the CAS12 enzyme cuts, it makes a staggered break in the DNA by about 5 nucleotides between cut sites. 
                              This is much preferred for DNA repair. It also cuts much further away from its PAM sequence. The next key 
                              domain of the CAS12 is the PAM sequence. This matches a specific sequence of the DNA. Each CAS enzyme has 
                              a different PAM site which it recognizes. The CAS12 PAM will recognize the sequence T-T-T-V. This stands 
                              for Thymine-Thymine-Thymine-Anything not another T. The CAS12 nucleases will cut about 18 to 23 bases away 
                              from its PAM sequence. This is important as this longer distance makes it good for CAS12 to do multiple 
                              edits. This CAS12 enzyme can sit at the PAM and make multiple edits with out disrupting its own activity.
                         </p>
                         <p>
                              The last part of the CAS12 system is the guide RNA.  The CAS12 enzyme does not require the tracrRNA. It only 
                              uses the crRNA which makes it much smaller for the guide RNA. The crRNA used by the CAS12 enzyme will only 
                              be about 42 nucleotides long. The guide RNA is installed in the delivery vector as RNA, but the CAS12 is 
                              often installed into the vector as a messenger RNA. Some of the newer RNP packaging will include the CAS12 
                              as a full enzyme. When its used as a mRNA, its about 3,800 bases in length and encodes a enzyme about 1,270 
                              amino acids long. This is still a very big package for most traditional vectors. Most companies have moved 
                              toward LNP technology for delivery.
                         </p>
                         <p>
                              The CAS12 will load the guide RNA when inside the cell. It enters the nucleus where it will bind 
                              to the DNA. The CAS9 opens the DNA and runs the 20 base guide along the DNA until it finds its 
                              match. Once the guide and PAM have a match, the nuclease will cut both DNA strands. Due to the fact that 
                              CAS12 makes a staggered cut of about 5 bases, it makes it a bit easier for the repair of the DNA without as 
                              many insertions and deletions (Indels).
                         </p>
                         <p>
                              This system can be used for Disruptions, Deletions or Insertions of genes. The disruption is done by causing 
                              Double Stranded Breaks at the target site to cause mutations that prevent reading of that gene. The second use is 
                              gene deletion. This can be created using 2 CAS12 enzymes along with 2 guides to attempt to cut and remove 
                              an entire section of the DNA. The last use is gene insertion using homology directed repair using a template 
                              strand that gets included with the CAS12 and guide RNA at the time of delivery. The CAS12 does the DNA 
                              cutting and the template strand guides the repair. One other use that CAS12 has found is in developing diagnostics. 
                              Companies use CAS12 to do diagnostics. This uses the CAS and guide as a search function to find DNA sequences. Then it 
                              cuts a reproter molecule which gives a readout.
                         </p>
                         <p>
                              The biggest concern with the CAS12 system is the Double Stranded Breaks. When DNA is cut like this, it can 
                              repair in unexpected ways. You can end up with insertions or deletions (Indels) of bases. The other big concern is 
                              chromosomal rearrangements when used for multiple edits at once. Typically they have to do an edit and run 
                              the cell through the cell cycle to ensure the first edit was good before they can do the next.
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="mad7">MAD7</h2>
                         <p>     
                              The MAD7 enzyme is a variant of the CAS12 which was found in bacteria from the Island of Madagascar. It shares 
                              a large amount of homology to the original CAS12 from which it most likely derived. The MAD7 system comes with 
                              2 main components with the guide RNA and the Nuclease. The MAD7 has only 1 nuclease with the RuvC just like 
                              with the CAS12 enzyme. It even makes the staggered double stranded cut like CAS12. The next key domain of the MAD7 
                              is the PAM sequence. This matches a specific sequence of the DNA. Each CAS enzyme has a different PAM site 
                              which it recognizes. The MAD7 PAM will recognize the sequence T-T-T-V. This stands for Thymine-Thymine-Thymine-Anything 
                              not another T. It also has the ability to recognize C-T-T-V which stands for Cytosine-Thymine-Thymine-Anything that is 
                              not another T. The early data form Inscripta shows both these PAM sequences work equally as well. The MAD7 nucleases 
                              will cut about 20 to 50 bases away from its PAM sequence. This is important as this longer distance makes 
                              it good for MAD7 to do multiple edits. This MAD7 enzyme can sit at the PAM and make multiple edits with out 
                              disrupting its own activity. The last part of the MAD7 system is the guide RNA.  The MAD7 enzyme does not require the 
                              tracrRNA. It only uses the crRNA which makes it much smaller for the guide RNA. This is the same as for the 
                              CAS12 enzyme. The MAD7 is just a few amino acids smaller than the CAS12 enzyme with about 1263 amino acids. This is about 
                              3,790 bases for a messenger RNA transcript.
                         </p>
                         <p>
                              The MAD7 system is owned and Licensed by Inscripta. They let anyone use the technology for free for 
                              research. They license it for a small royalty for commercial use. I know a few companies that license it 
                              from them. It works pretty much like CAS12 with some minor differences. 
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="casx">CASX</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="cas13">CAS13</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="mini">CAS14 and Phi</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="base">Base Editing</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="prime">Prime Editing</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

                    <section>
                         <h2 class="science-headers" id="paste">PASTE Editing</h2>
                         <p>     
                              
                         </p>
                         <p class="top-link"><a href="#top">Top</a></p>
                    
                    </section>

               </div>

          </main>

          <!--Here Starts the Footer-->

          <footer id="footer">
               <div>
                    <a href="https://www.twitter.com/biotech2k1">
                         <img class="twitter-icon" src="../images/twitterlogo.jpg" alt="Follow us on Twitter">
                    </a>
               </div>
               <div>
                    <p class="follow-us">Follow us on Twitter</p>
               </div>
          </footer>

          <!--Javascript-->

          <script src="../code/header.js"></script>

     </body>
</html>